src/core/gc/config.d

```diff
@@ -55,6 +55,8 @@ struct Config
     parallel:N     - number of additional threads for marking (%lld)
     heapSizeFactor:N - targeted heap size to used memory ratio (%g)
     cleanup:none|collect|finalize - how to treat live objects when terminating (collect)
+
+    Memory-related values can use B, K or M suffixes.
```

Comment:
I'd rather meant that the displayed values don't match the shown unit. For bonus points, it would be nice to show the suffix if possible for exact values (e.g. 0x4000 could show 16K).

Reply:
Ah, totally forgot about this

> For bonus points, it would be nice to show the suffix if possible for exact values (e.g. 0x4000 could show 16K).

I don’t plan to code in the next couple of days, maybe implement this in a separate PR?

Reply:
> For bonus points, it would be nice to show the suffix if possible for exact values (e.g. 0x4000 could show 16K)

Done